src/pages/List/List.js

```diff
@@ -33,6 +33,7 @@ class List extends Component {
               {this.state.beverageData.map(product => (
                 <ListBeverageCard
                   key={product.id}
+                  id={product.id}
                   img={product.img}
                   name={product.name}
```

Comment:
같은 이름으로 props로 넘기려면 아래처럼 하시면 됩니다!
```suggestion
   {...product}
```

Reply:
오호 이런방법이 있었군요!👍
감사합니다! 다음번에 비슷한 상황이 생긴다면 한 번 적용해 봐야겠어요 🙂!